train_tacotron.py

```diff
@@ -167,25 +179,19 @@ def create_align_features(model: Tacotron,
     train_cfg = config['tacotron']['training']
     if args.force_gta:
```

Comment:
There is no check for passed arguments, for example one can pass extract_pitch and force_gta and after extracting pitch the process will finish. It creates some confusion to have training as one of the options in the if-else block with other operations. I would suggest to keep only training logic in this script, and move force_align force_gta extract_pitch to another helper script with own arguments and execution flow

Reply:
I agree but I would like to do that in a separate refactoring - otherwise the changes are getting even larger.